code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -46,6 +46,47 @@
 	ammo_type = /obj/item/ammo_casing/c45
 	max_ammo = 30
 
+
+/obj/item/ammo_magazine/c380
+	name = "ammunition Box (.380)"
+	icon_state = "9mm"
+	origin_tech = list(TECH_COMBAT = 2)
+	caliber = ".380"
+	matter = list(DEFAULT_WALL_MATERIAL = 2250)
+	ammo_type = /obj/item/ammo_casing/a380
+	max_ammo = 30
+
+
+/obj/item/ammo_magazine/c380t
+	name = "ammunition Box (.380 T)"

```

Comment:
Why this exists? It seems ot be the same as the magazine.

Reply:
Its a box of bullets

Reply:
It will work as a magazine anyway, since it is one, remove those.